packages/semver/src/schematics/ng-add/schema.json

```diff
@@ -13,6 +13,18 @@
     "projects": {
       "description": "Which projects should be versioned independently",
       "type": "array"
+    },
+    "enforceConventionalCommits": {
+      "description": "Install & configure commitlint, husky & commitizen",
+      "type": "boolean",
+      "default": false,
```

Comment:
I would suggest setting it to `true` by default.

Reply:
I agree, done